src/components/AddNewPage/AddNewPage.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components/macro'
+import Card from '../Card/Card'
+import AddNew from '../AddNew/AddNew'
+
+export default function AddNewPage({ addNewCard, cards, setCards }) {
+  return (
+    <AddNewPageLayout>
+      <AddNew onAddNewCard={addNewCard} />
+      <ResultContainer>
+        {cards.map(card => (
+          <Card
+            name={card.name}
+            store={card.store}
+            price={card.price}
+            date={card.date}
+            clothingType={card.clothingType}
+            images={card.images}
+            cards={cards}
+            setCards={() => setCards()}
+          />
+        ))}
+      </ResultContainer>
+    </AddNewPageLayout>
+  )
+}
+const AddNewPageLayout = styled.div`
```

Comment:
Suggestion:
If I have understood your code correctly, you could use `main` instead of `div` for more semantic HTML